### PR TITLE
Run celery also on openshift instance

### PR DESCRIPTION
**Pull request by alesrosina**
Covers issues https://github.com/WeblateOrg/weblate/issues/2808 and https://github.com/WeblateOrg/weblate/issues/372

### CODE REVIEW

**Comment by alesrosina** [comment]
@nijel & @GrahamDumpleton a code review would be appreciated.

But this solution seems to work.

File: weblate/settings_openshift.py [hunk 1]
```diff
@@ -94,6 +94,29 @@
...
+    CELERY_BEAT_SCHEDULE_FILENAME = os.path.join(
+        DATA_DIR, 'celery', 'beat-schedule'
+    )
+    CELERY_TASK_ROUTES = {
+        'weblate.trans.search.*': {'queue': 'search'},
+        'weblate.trans.tasks.optimize_fulltext': {'queue': 'search'},
+        'weblate.trans.tasks.cleanup_fulltext': {'queue': 'search'},
+        'weblate.memory.tasks.*': {'queue': 'memory'},
+        'weblate.accounts.tasks.notify_change': {'queue': 'notify'},
+    }
```
Comment (nijel) [inline weblate/settings_openshift.py]: You already get the above 4 settings from settings_example, so better not to copy them.

Comment (alesrosina) [reply weblate/settings_openshift.py]: Right, I've removed this. Thanks.

**Comment by nijel** [comment]
Merged, thanks for your contribution!

**Pull request merged**